angelspetrochem/prototypes/buildings/oil-refinery.lua

```diff
@@ -35,7 +35,7 @@ data:extend({
     }, 2, angelsmods.petrochem.number_tint),
     flags = { "placeable-neutral", "player-creation" },
     -- Base-game oil refinery has a custom collision_mask, which all upgrades need to match.
-    collision_mask = {layers={item=true, object=true, player=true, water_tile=true, elevated_rail=true, is_object=true, is_lower_object=true}},
+    collision_mask = oilrefinery.collision_mask,
```

Comment:
I didn't know you could reference the base-game entity like this?

Reply:
well, I thought it would be a bit tedious to keep track of the changes to the `collision_mask` of the entity every time, so I tried just using `collision_mask` straight from the base entity and it worked

Reply:
Fair! Wish I'd thought of that haha